pricelist_per_product/static/description/index.html

```diff
@@ -0,0 +1,48 @@
+
+<section class="oe_container oe_dark">
+    <div class="oe_row oe_spaced">
+            <h2 class="oe_slogan">Pricelist Configuration</h2>
+            <h3 class="oe_slogan">Product Price Grid</h3>
+        <div class="oe_span6">
+            <p class="oe_mt32">
+Define in the pricelist if the price items are fulfilled from product form
+with a grid of specific values for each product.
+            </p>
+        </div>
+        <div class="oe_span6">
+            <img class="oe_picture oe_screenshot" src="pricelist.png">
+        </div>
+    </div>
+</section>
+
+
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <h2 class="oe_slogan">Pricelist Grid</h2>
+        <h3 class="oe_slogan">All pricelists at once</h3>
+        <div class="oe_span6">
+            <img class="oe_picture oe_screenshot" src="product.png">
+        </div>
+        <div class="oe_span6">
+            <p class="oe_mt32">
+Fulfill all your pricelists directly in your product form.
```

Comment:
s/Fullfill/Fill in/

to fullfill c'est "remplir" au sens de "remplir une obligation", "satisfaire une condition"